frontend/src/metabase/entities/actions/forms.ts

```diff
@@ -0,0 +1,41 @@
+import { t } from "ttag";
+
+const getFormFields = (formAction: "create" | "update") => [
+  {
+    name: "name",
+    title: t`Name`,
+    placeholder: t`My new fantastic action`,
+    autoFocus: true,
+    validate: (name: string) =>
```

Comment:
- Shouldn't that be handled on the schema level? Or would you use that function later when creating the schema?
- WDYT about using error message helpers here? from `utils/errors`

Reply:
This whole file is going to be deleted in #27673 where we migrate new action form from entity forms to direct formik usage